generators/client/templates/_package.json

```diff
@@ -8,6 +8,7 @@
     "<%= MAIN_SRC_DIR %>bower_components"
   ],
   "devDependencies": {
+    "bower": "1.7.9",
```

Comment:
didn't know about the react module. will need to check that out ....

Did some testing with / without the bower dependency in package.json and couldn't really see an impact on build time (timed an npm install without node_modules to force full download)

**with bower**
05:47
05:56

**without bower**
05:43
05:48

As the CI profile would require bower in package.json, can we meet half-way and still keep it in package.json so that we can support both options :

1. people who want to use global bower/gulp modules can keep on executing them directly on the command line. 
2. people that opt not to install global npm modules can execute bower/gulp via the plugin or [npm scripts](https://docs.npmjs.com/misc/scripts) on the cmdline (as these npm scripts can access the devDependencies)